hadoop-project/src/site/markdown/index.md.vm

```diff
@@ -15,103 +15,143 @@
 Apache Hadoop ${project.version}
 ================================
 
-Apache Hadoop ${project.version} is an update to the Hadoop 3.3.x release branch.
+Apache Hadoop ${project.version} is an update to the Hadoop 3.4.x release branch.
 
 Overview of Changes
 ===================
 
 Users are encouraged to read the full set of release notes.
 This page provides an overview of the major changes.
 
-Azure ABFS: Critical Stream Prefetch Fix
+S3A: Upgrade AWS SDK to V2
 ----------------------------------------
 
-The abfs has a critical bug fix
-[HADOOP-18546](https://issues.apache.org/jira/browse/HADOOP-18546).
-*ABFS. Disable purging list of in-progress reads in abfs stream close().*
+[HADOOP-18073](https://issues.apache.org/jira/browse/HADOOP-18073) S3A: Upgrade AWS SDK to V2
 
-All users of the abfs connector in hadoop releases 3.3.2+ MUST either upgrade
-or disable prefetching by setting `fs.azure.readaheadqueue.depth` to `0`
+This release of Hadoop moves the S3A connector to Amazon S3 to the V2 SDK.
```

Comment:
This release of Hadoop moves the S3A connector to Amazon S3 to the V2 SDK.
->
This release upgrade Hadoop's AWS connector S3A from AWS SDK for Java V1 to AWS SDK for Java V2.

Reply:
Thanks for the suggestion! I will improve it.